server/organizations.go

```diff
@@ -56,17 +57,13 @@ func (r *organizationRequest) ValidDefaultRole() error {
 }
 
 type organizationResponse struct {
-	Links       selfLinks `json:"links"`
-	ID          uint64    `json:"id,string"`
-	Name        string    `json:"name"`
-	DefaultRole string    `json:"defaultRole,omitempty"`
+	Links selfLinks `json:"links"`
+	chronograf.Organization
```

Comment:
this works, but we favor explicit definitions in the rest of the codebase -- please to change to explicit definition, despite how good sugar tastes. we try to have our structs be distinct from our responses.

Reply:
👎